### PR TITLE
Added Bat file

### LINKED ISSUE #56: Setup Linkees
Can you add the whole setup? Because new coders can also know how to setup the project from the start to end.

### PR DESCRIPTION
**Pull request by Bashamega**
fixes #56

### CODE REVIEW

**Comment by heysagnik** [comment]
@taksuparth can you please review it

**Comment by taksuparth** [comment]
yes @heysagnik, checking

File: setup.bat [hunk 1]
```diff
@@ -0,0 +1,24 @@
+@echo off
+set "zipUrl=https://github.com/heysagnik/Linkees/archive/refs/heads/old.zip"
```
Comment (taksuparth) [inline setup.bat:2]: How will the user receive updated zip files?

File: setup.bat [hunk 2]
```diff
@@ -0,0 +1,24 @@
+@echo off
+set "zipUrl=https://github.com/heysagnik/Linkees/archive/refs/heads/old.zip"
+set "zipFile=old.zip"
+set "extractDir=Linkees-old"
```
Comment (taksuparth) [inline setup.bat:4]: folder name should be Linkees

File: setup.bat [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+)
+
+echo Extracting %zipFile%...
+powershell -command "Expand-Archive -Path .\%zipFile% -DestinationPath '.'"
+if exist %zipFile% (
+    del %zipFile%
+)
+ren "Linkees-old" "Project"
+cd Project
+npm install
```
Comment (taksuparth) [inline setup.bat:21]: This assumes that npm is available

File: setup.bat [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
```
Comment (taksuparth) [inline setup.bat:22]: This doesn't give user easy access to update the information on landing page. Moreover, why we are not setting up an app and starting it?

File: setup.bat (hunk 1 above)
Comment (Bashamega) [reply setup.bat:2]: Would you like it to be the master branch?

File: setup.bat (hunk 2 above)
Comment (Bashamega) [reply setup.bat:4]: ok

File: setup.bat (hunk 3 above)
Comment (Bashamega) [reply setup.bat:21]: ok

File: setup.bat (hunk 4 above)
Comment (Bashamega) [reply setup.bat:22]: would you like it to start the server?

**Comment by Bashamega** [comment]
hello:)
I have made it save as Linkees but I could add the yarn part sorry.
@taksuparth

**Comment by Bashamega** [comment]
@taksuparth @heysagnik When will you merge?

**Pull request merged**